llm/memory.go

```diff
@@ -70,7 +71,7 @@ type MemoryEstimate struct {
 
 // Given a model and one or more GPU targets, predict how many layers and bytes we can load, and the total size
 // The GPUs provided must all be the same Library
-func EstimateGPULayers(gpus []discover.GpuInfo, ggml *GGML, projectors []string, opts api.Options) MemoryEstimate {
+func EstimateGPULayers(gpus []discover.GpuInfo, f *ggml.GGML, projectors []string, opts api.Options) MemoryEstimate {
```

Comment:
Is there a better name than `f` here? the `f` confused me with `os.File` below

Reply:
Maybe it's more the type in `ggml` - e.g. `m *ggml.Model` might be clearer

Reply:
It's not a model. `File` is a better name since it represents a ggml file